app/utils/timeHelpers.ts

```diff
@@ -1,3 +1,4 @@
+import { setHours, setMinutes, setSeconds, setMilliseconds } from 'date-fns';
```

Comment:
Can we just use functions from a transitive dependency? :thinking:

Reply:
You can, but it's obviously pretty bad to do.. I'll make sure they're installed.